### DISCUSSION TITLE
Message log option

### DISCUSSION THREAD

**Discussion by mstick99** [lart2150/kmttg#30] [Ideas]
Selectable option to automatically create the event log of messages rather than having to use menu (File, save messages to file). Maybe along the other selectable options somewhere after the START JOBS button.

**Comment by lart2150** [comment]
This is a interesting idea.  Are you thinking this would always replace kmttg.log or write to something like kmttg-yyyy-mm-dd-hh-mm-ss.log?  I think it would be a much larger change but more useful to separate the logs for each job aka if I kick off 3 transfers and wanted each transfer to have it's own log.  If each transfer had it's own log we could put the log file with the output file kindof like the metadata file.

**Comment by mstick99** [comment]
I was initially thinking of the first method as it would automatically be created for review of kmttg download errors. Maybe include the date of the kmttg run. The second method would be even more useful, especially if for jobs with an error the log name included the phrase error -- something like kmttg-yyyy-mm-dd-hh-mm-ss-error.log.
